content/webapp/services/prismic/types/articles.ts

```diff
@@ -40,7 +40,7 @@ export const eventsFetchLinks: FetchLinks<EventPrismicDocument> = [
   'events.schedule',
   'events.interpretations',
   'events.series',
-  'events.times',
+  'events.locations',
   'events.place',
```

Comment:
should this change be `place` instead of `times`?

Reply:
Well if you will be fussy and insist it's _correct_! :D

Reply:
sorry to be soooooo finicky